starfive/visionfive/v2/default.nix

```diff
@@ -18,4 +13,19 @@
       generic-extlinux-compatible.enable = lib.mkDefault true;
     };
   };
+
+  hardware.deviceTree.name =
+    lib.mkDefault "starfive/jh7110-starfive-visionfive-2-v1.3b.dtb";
+
+  hardware.deviceTree.overlays = [{
+    name = "qspi-patch";
+    dtsFile = ./qspi-patch.dts;
```

Comment:
Is this file missing in your commit?

Reply:
The entire `hardware.deviceTree` config can be dropped. They all have working defaults with recent u-boot/kernel.